geomstats/geometry/connection.py

```diff
@@ -660,3 +660,21 @@ def parallel_transport(
             "The closed-form solution of parallel transport is not known, "
             "use the ladder_parallel_transport instead."
         )
+
+    def injectivity_radius(self, base_point):
+        """Compute the radius of the injectivity domain.
+
+        This is the largest radius r for which the exponential map is a
+        injective on the open ball of radius r and centered at the base point.
+
```

Comment:
I think the original definition of the injectivity radius is the supremum of radiuses r for which the exponential map is a diffeomorphism from the open ball of radius r centered at the base point onto its image. I don't know if it is equivalent to "largest radius for which the exponential map is injective".